### PR TITLE
2021-12-29-classifierdl_urduvec_fakenews_ur

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-29-classifierdl_urduvec_fakenews_ur.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+classifierdl = ClassifierDLModel.pretrained("classifierdl_urduvec_fakenews", "ur") \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+urdu_fake_pipeline = Pipeline(stages=[document_assembler, tokenizer, normalizer, lemma, embeddings, embeddingsSentence, classifierdl])
+
+light_pipeline = LightPipeline(urdu_fake_pipeline.fit(spark.createDataFrame([['']]).toDF("news")))
+
+result = light_pipeline.annotate("پاکستان کے اخبارات روزانہ اس بات کی مثالیں دیتے ہیں کہ کس طرح مصنوعی ذہانت کو ذاتی مقاصد کے لیے استعمال کیا جا سکتا ہے۔")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-29-classifierdl_urduvec_fakenews_ur.md:71]: Two things:
1) Is this a real text? I can't find it in the Internet. What is the source? If it's a translation, use real examples from Wikipedia.
2) Why is this fake? 
image

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can you take a look at the requested changes? Thank you!

File: docs/_posts/murat-gunay/2021-12-29-classifierdl_urduvec_fakenews_ur.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-29-classifierdl_urduvec_fakenews_ur.md:71]: @josejuanmartinez 
I took it from the train data-set. 
There can't be a county whose all newspapers give daily news regarding AI for personal purposes. I don't see any news regarding AI uses for personal purposes. Besides, it happens daily.... everyday... impossible, I guess.

**Comment by josejuanmartinez** [comment]
That's what makes the example confusing, that who knows, If Pakistan is that country, or not. Also, what if tomorrow they start to write about AI? It's non deterministic at all. Please, chose something that is evident for the users. Thank you!

**Review by josejuanmartinez — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**